conf/test_busco_auto.config

```diff
@@ -16,8 +16,8 @@ params {
 
     // Limit resources so that this can run on GitHub Actions
     max_cpus   = 2
-    max_memory = 6.GB
```

Comment:
This has to be specified on the CLI I thought?

Reply:
AS otherwise people wanting to use the test profile on GCP/AWS will not be able to use them

Reply:
line 8 is the "Usage:", I do think it would be good to mention otherwise there will be an error when starting the test with the command under "Usage:".

Reply:
Ah understood, my bad.